enterprise/server/vmexec/vmexec.go

```diff
@@ -17,6 +17,14 @@ import (
 	vmxpb "github.com/buildbuddy-io/buildbuddy/proto/vmexec"
 )
 
+const (
+	// workspaceDevice is the path to the hot-swappable workspace block device.
+	workspaceDevice = "/dev/vdc"
+	// workspaceMountPath is the path where the hot-swappable workspace block
+	// device is mounted.
+	workspaceMountPath = "/workspace"
```

Comment:
If these need to match what's in init, I would note that and link to the source.

Reply:
Done